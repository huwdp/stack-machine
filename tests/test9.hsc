pushf 1.3
printl
toi
printl
pop
pushi 2000
tof
printl