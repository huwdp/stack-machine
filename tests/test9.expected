Welcome to Huw's Stack Machine
1.3
1
2000
